config.go

```diff
@@ -61,8 +61,10 @@ type BuildStep struct {
 
 // Tool describes a project to build and test.
 type Tool struct {
-	// Name has the tool name that is the same as the executable name.
+	// Name has the tool name.
 	Name string
+	// BinaryName has the executable name of the tool.
+	BinaryName string
```

Comment:
does it make sense to keep backward compatibility and use `Name` if `BinaryName` is empty?

Reply:
+1 to keep backward compatibility

Reply:
Yeah it totally makes sense.

Reply:
Addressed [here](https://github.com/src-d/regression-core/pull/9/commits/dd30bc1e7cb9026089c6c3ad78ef4606ea1691c8).